src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentSubjects.js

```diff
@@ -28,10 +28,10 @@ export class DocumentSubjects extends Component {
       }
       return <MetadataTable rows={rows} />;
     }
-    return null;
+    return 'There is no subject classification information.';
```

Comment:
Maybe just me, but I find weird that `There is no` at the beginning...
```suggestion
    return 'No subject classification information.';
```